index.bs

```diff
@@ -10183,44 +10226,25 @@ Constructors</h5>
 		the following steps are performed on the <a>rendering thread</a>.
 
 		<div algorithm="AudioWorkletProcessor()">
-			This constructor is invoked by processing a control
-			message queued by the {{AudioWorkletNode()|constructor}}
-			of the associated {{AudioWorkletNode}}. This control
-			message carries the [=processor construction data=] that
-			consists of:
-			<var>nodeName</var>,
-			<var>serializedProcessorPort</var>,
-			<var>serializedOptions</var>,
-			and <var>node</var>.
-
 			If any of these steps throws an exception, abort the rest of
 			steps and <a>queue a task</a> to fire an
 			<a href="https://www.w3.org/TR/html50/webappapis.html#the-errorevent-interface">ErrorEvent</a>
 			named <code>processorerror</code> at the {{AudioWorkletNode}}
 			on the <a>control thread</a> with the relevant information about
 			the error.
 
-			1. If there is no [=processor construction data=]
-				passed from the associated {{AudioWorkletNode}},
-				throw a {{TypeError}}.
-
-			1. Let <var>processorPort</var> be
-				[$StructuredDeserializeWithTransfer$](<var>serializedProcessorPort</var>,
-				the current Realm).
+			1. If there is no provided [=nodePort=] at the
+				invocation of the constructor throw a
+				{{TypeError}}.
 
-			1. Let <var>options</var> be
-				[$StructuredDeserialize$](<var>serializedOptions</var>,
-				the current Realm).
+			1. Let <var>processor</var> be a new
+				{{AudioWorkletProcessor}} instance.
```

Comment:
You don't want to do this.  This part is handled for you by IDL already; see https://heycam.github.io/webidl/#create-an-interface-object step 1.8 and 1.9.  You can refer to that object here as "this".